src/thucal2.iced

```diff
@@ -470,6 +474,29 @@ unsafeWindow.thucal=thucal=new ->
       @ui.log '不支持小学期！'
       return
 
+    if document.location.toString().match(/sslvpn/)
+      @ui.log "SSLVPN 模式: 自动登录info..."
+      #open info page (to utilize sslvpn system's auto-login feature)
```

Comment:
I know I'm stupid, it's for testing only at that time!

let's change that to `document.location.host.match` ?